src/app/components/Promotions/index.tsx

```diff
@@ -22,59 +22,60 @@ export const Promotions: React.FC = () => {
           <PromotionCard
             appSection={AppSection.YieldFarm}
             backgroundColor={PromotionColor.Orange}
-            title="5K SOV rewards"
-            duration="Ongoing weekly rewards"
-            text="Provide a 1:1 ratio of MYNT and rBTC to the MYNT/BTC AMM liquidity pool and instantly start accruing your share of 5,000 SOV rewards."
-            learnMoreLink="https://www.sovryn.app/blog/sovryn-mynt-project-updates"
+            title={t(translations.promotions.card1.title)}
+            duration={t(translations.promotions.card1.duration)}
+            text={t(translations.promotions.card1.text)}
+            learnMoreLink="https://wiki.sovryn.app/en/sovryn-dapp/market-making#yield-farming"
```

Comment:
I would extract the link to `utils/classifiers`.

Reply:
done